library/constant_time_impl.h

```diff
@@ -123,7 +123,7 @@ static inline mbedtls_ct_uint_t mbedtls_ct_compiler_opaque(mbedtls_ct_uint_t x)
  * For gcc, restore divided syntax afterwards - otherwise old versions of gcc
```

Comment:
```suggestion
 * For old versions of gcc excluding 4.8 and 4.9 (see #8516 for details), restore divided syntax afterwards - otherwise old versions of gcc
```